library/include/modules/Units.h

```diff
@@ -138,9 +138,11 @@ DFHACK_EXPORT std::string getRaceBabyName(df::unit* unit);
 DFHACK_EXPORT std::string getRaceChildNameById(int32_t race_id);
 DFHACK_EXPORT std::string getRaceChildName(df::unit* unit);
 
+DFHACK_EXPORT bool isInvader(df::unit* unit);
```

Comment:
Could you add the new functions to `dfhack_units_module` in LuaApi.cpp and document them in Lua API.rst?

Reply:
Dang, you mean they don't auto-export to lua

Reply:
exported.